### PR TITLE
fixing broken links

### PR DESCRIPTION
**Pull request by seankane-msft**
Fixing broken links found by the [python - aggregate - reports](https://dev.azure.com/azure-sdk/internal/_build/results?buildId=777916&view=logs&j=0874ffeb-2919-5b4b-20de-16a97970b94c&t=3e781ea0-2593-5805-56b9-073e5359e33c) pipeline

### CODE REVIEW

**Comment by yunhaoling** [comment]
I remember `https://azure.github.io/azure-sdk/python_introduction.html` is still available on Monday or Tuesday.. other the build pipeline for the release would fail..

maybe we should check with the EngSys team to see why `python_introduction.html` is removed this

**Review by johanste — APPROVED** [review]

File: sdk/keyvault/azure-keyvault-certificates/migration_guide.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ To try and improve the development experience across Azure services, a set of un
 
 ### Cross Service SDK improvements
 
-The modern Key Vault client library also provides the ability to share in some of the cross-service improvements made to the Azure development experience, such as 
+The modern Key Vault client library also provides the ability to share in some of the cross-service improvements made to the Azure development experience, such as
```
Comment (mccoyp) [inline sdk/keyvault/azure-keyvault-certificates/migration_guide.md:30]: I updated these links yesterday but didn't realize that everyone had the same issue -- on line 26 I used https://azure.github.io/azure-sdk/general_introduction.html, but that can be replaced with https://azure.github.io/azure-sdk/python/guidelines/index.html. (This goes for keys and secrets as well)

Comment (seankane-msft) [reply sdk/keyvault/azure-keyvault-certificates/migration_guide.md:30]: Adding this change

**Review by mccoyp — APPROVED** [review]
Thank you! :shipit:

**Comment by mitchdenny** [comment]
Looks like this PR caused a CI failure, could you investigate @seankane-msft:
https://dev.azure.com/azure-sdk/public/_build/results?buildId=779018&view=results

**Comment by seankane-msft** [comment]
> Looks like this PR caused a CI failure, could you investigate @seankane-msft:
> https://dev.azure.com/azure-sdk/public/_build/results?buildId=779018&view=results

@mitchdenny The failure is a test in Python 2.7, I don't believe it has to do with the changed links here.

**Pull request merged**